icgcget/clients/pdc/pdc_client.py

```diff
@@ -29,43 +29,44 @@ def __init__(self, pickle_path=None):
         super(PdcDownloadClient, self).__init__(pickle_path)
         self.repo = 'pdc'
 
-    def download(self, data_paths, access, tool_path, output, processes, udt=None, file_from=None, repo=None,
-                 region=None):
-        access_file = open(access)
-        key = access_file.readline()
-        secret_key = access_file.readline()
-        os.environ['AWS_ACCESS_KEY_ID'] = key.rstrip()
-        os.environ['AWS_SECRET_ACCESS_KEY'] = secret_key.rstrip()
-        os.environ['AWS_DEFAULT_REGION'] = region
+    def download(self, data_paths, key, tool_path, output, processes, udt=None, file_from=None, repo=None,
+                 secret_key=None):
+        code = 0
+        os.environ['AWS_ACCESS_KEY_ID'] = key
+        os.environ['AWS_SECRET_ACCESS_KEY'] = secret_key
         for data_path in data_paths:
-            call_args = [tool_path, 's3', 'cp', data_path, output + '/']
-            self._run_command(call_args, self.download_parser)
+            call_args = [tool_path, 's3', '--endpoint-url=https://bionimbus-objstore.opensciencedatacloud.org/', 'cp',
```

Comment:
`https://bionimbus-objstore.opensciencedatacloud.org` should be a constant in this class.